.github/workflows/ci.yaml

```diff
@@ -0,0 +1,73 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - "*.*"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '8'
```

Comment:
11?

Reply:
didn't want to change it in this branch.. will have that as a separate PR.